volontulo/templates/common/header.html

```diff
@@ -28,12 +28,12 @@ <h1 class="navbar-header">
                 </li>
                 {% else %}
                 <li class="dropdown">
-                    <a class="dropdown-toggle" href="#" aria-expanded="false" aria-haspopup="true" role="button" data-toggle="dropdown">
+                    <a class="dropdown-toggle" href="{% url 'logged_user_profile' %}" aria-expanded="false" aria-haspopup="true" role="button" data-toggle="dropdown">
```

Comment:
I don't get it. For now I only needed link to redirect to users profile page, not complete/functional dropdown. We don't even know if here will be any dropdown.